src/lib.rs

```diff
@@ -54,347 +58,456 @@ impl Default for Duration {
     }
 }
 
-#[derive(Serialize, Debug, Builder)]
-#[builder(setter(into))]
-#[serde(rename_all = "camelCase")]
-pub struct ChartParams {
-    #[builder(default)]
-    pub chart_reset: Option<bool>,
-    #[builder(default)]
-    pub chart_simplify: Option<bool>,
-    #[builder(default)]
-    pub chart_interval: Option<i64>,
+#[derive(PartialEq, Eq)]
+/// The `Request` enum type allows for HTTP request matching to the IEX API.
+// TODO(Request): Add documentation from IEX website.
```

Comment:
I think just a link to the relevant part of the documentation would suffice. No need to add it in this PR, though.